docs/content/concepts/sui-move-concepts/packages/automated-address-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Automated Address Management
+---
+
+When you publish or upgrade a package, its address (also known as the package ID) is tracked in the `Move.lock` file. This bookkeeping is done automatically so that you can avoid recording or updating hex addresses (e.g., in the `Move.toml` file). 
+
+When you publish or upgrade your package on multiple chains (e.g., `mainnet`, `testnet`, `devnet`) separate addresses for each chain will be tracked for you. This tracking is based on your active environment (e.g., run `sui client active-env`). For example, if you set your active environment to an RPC connected to `testnet` and publish a package, the `Move.lock` will record the address for that package and associate it with your active environment (`testnet`).
+
+Note that automated address management works for one package published to one or more chains. When you publish or upgrade, it uses the contents of the package's working directory. If a package is republished to a chain, addresses tracked for the previously published package will be overwritten for that chain.
+
+## Adopting automated address management for published packages
+
+Previously a `published-at` entry was **mandatory** in the `Move.toml` file, which sets the address of the latest published package. It is no longer required if this data is tracked in the `Move.lock` file. For an existing package, you'll need to add the necessary data to the `Move.lock` file so that it can be automatically tracked:
+
+1. Switch to your active environment for the chain that your package is published on
+
```

Comment:
```suggestion
```
These next several suggestions should make the nested list and nested code work better. As is, it's creating separate `ol` elements that can mess up numbering if more items are added later. Basically, anything that should go under the list item have to be indented (two tabs worth usually gets it).